README.md

```diff
@@ -1,172 +1,124 @@
+all experiments will be logged to the folder `mlruns` created in the
+directory from which the script is launched. To then look at the
+experiments in the browser we just need to run the command `mlflow
+ui`.
 
-We use [Sphinx](https://www.sphinx-doc.org/) to generate documentation to our
-projects. The documentation files can be written using either Markdown or
-reStructuredText syntax and are stored in the `docs` directory. See
-[`docs/index.md`](docs/index.md) for a collection of simple examples to get you
-started including math formulas and code snippets in your documentation.
+## Installing everything
 
-If you want to generate the documentation locally, simply install the
-dependencies via `pip`:
+The next step is to actually clone the repository using:
 
 ```bash
-pip install -r docs/requirements.txt
+git clone https://github.com/inductiva/meshnets.git
 ```
 
-and run
+The very next step is to create a virtual environment. This will solve
+any clashes with the library versions used here and anything else that
+might be installed in your own system:
 
 ```bash
-sphinx-build -b html docs docs/_build
+python3 -m venv .env
+source .env/bin/activate
 ```
 
-in your base directory to build the documentation in `docs/_build`.
-
-To serve the documentation locally and automatically build it with new changes,
-install the `sphinx-autobuild` package,
+After creating and activating the virtual environment we can install
+all the requirements of the project using:
 
 ```bash
-pip install sphinx-autobuild
+pip install --upgrade pip
+pip install -r requirements.txt
 ```
 
-and run in your base directory
+Next, because `meshents` is actually packaged we can install it using:
 
 ```bash
-sphinx-autobuild -b html docs docs/_build
+pip install -e .
 ```
 
-This will serve the documentation at <http://127.0.0.1:8000> and automatically
-update it when you save changes to the documentation files.
+## Training the models
 
-The documentation may also be made available on
-[Read The Docs](https://readthedocs.org/) for configured repositories. When set
-up, this integration will build and deploy the documentation on each commit to
-the `main` branch.
+Now to the fun part. Training a model is simply a matter of running
+the script `train.py`:
 
-See the documentation for the start-a-project repo at:
-
-https://inductiva-research-labs-start-a-project.readthedocs-hosted.com/en/latest/
-
-Apart from documentation files, we also prepare README files using Markdown
-syntax to present short introductions of the projects. This README is one such
-example.
-
-
-## Conda Environments
-
-To automate all the setup commands above, we have created a Conda
-environment config file that you can use for your convenience.
-
-Make sure the first line of the conda.yml file contains a name related to 
-your project, such as:
-
-```
-name: startaproject_env
+```python
+python train.py --data_dir=<Path to your dataset>
```

Comment:
If I follow all of these steps will this actually work? Notice, that we haven't said anywhere to split the datasets.

Reply:
![image](https://github.com/inductiva/meshnets/assets/33664950/a153b36e-2dba-46cf-bfe0-bec8b20db7ca)
It is split in the main script.

Reply:
That the path to the validation dataset is the name of the folder that then stays with the validation samples, right?